config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_logginglogmetrics.logging.cnrm.cloud.google.com.yaml

```diff
@@ -80,31 +79,27 @@ spec:
                     properties:
                       growthFactor:
                         description: Must be greater than 1.
-                        format: double
                         type: number
                       numFiniteBuckets:
                         description: Must be greater than 0.
-                        format: int64
+                        format: int32
```

Comment:
I don't think down-scaling is technically safe. I guess we can make an argument that the number of buckets must be less than 2,147,483,647.